Jenkinsfile

```diff
@@ -23,10 +23,10 @@ pipeline {
 			steps {
 				sh """
 				mvn clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
-					-Pbuild-individual-bundles -Pbree-libs -Papi-check -Pjavadoc\
+					-Papi-check -Pjavadoc\
 					-Dcompare-version-with-baselines.skip=false \
 					-Dproject.build.sourceEncoding=UTF-8 \
```

Comment:
```suggestion
```
The `sourceEncoding` is specified in the parent-pom as well and therefore can be removed.